cdap-ui/app/cdap/components/DataPrepHome/index.js

```diff
@@ -266,7 +288,7 @@ export default class DataPrepHome extends Component {
       );
     }
 
-    if (this.state.backendDown) {
+    if (this.state.backendDown || !this.state.isMinVersionMet) {
```

Comment:
The error should be surfaced here right? If the min version is not met will the user will still see the enable button?

Reply:
This is not an error per se. It's about the running dataprep service does not meet the minimum version required by the front end. So we are treating it as if the service is not enabled. Enabling it will recreate the app using the correct artifact version, or if the minimum artifact version not found, it will error out at this stage.